### DISCUSSION TITLE
New Wiki online!

### DISCUSSION THREAD

**Discussion by mr-manuel** [Louisvdw/dbus-serialbattery#543] [General]
Now the new Wiki is online! Check it out &rarr; https://louisvdw.github.io/dbus-serialbattery/

What are your thoughs?

**Comment by gnissahr** [comment]
Excellent! From now on RT(F)M will be a valid reply to a lot of questions asked ;-)

**Reply by mr-manuel** [reply]
Yes, at all with the changes of today
